Changelog for 3.0.0 release (#844)
vivid-planet/comet: CHANGELOG.md

-   Rename `FormSectionKey` to `FormSectionClassKey`
-   Remove the default clear button from `FinalFormSearchTextField`. Re-enable by using the `clearable` prop

#### Changes

-   Expose `comet.generic.*` messages as public API through `messages.ts`. Doing so prevents the need to translate them for every application:

    **Before**

    ```tsx
    <FormattedMessage id="comet.generic.globalContentScope" defaultMessage="Global Content" />;

    intl.formatMessage({
        id: "comet.generic.doYouWantToSaveYourChanges",
        defaultMessage: "Do you want to save your changes?",
    });
    ```

    **After**

    ```tsx
    import { messages } from "@comet/admin";

    <FormattedMessage {...messages.globalContentScope} />;

    intl.formatMessage(messages.saveUnsavedChanges);
    ```

-   `StackBreadcrumbs` now shows an overflow menu when its items do not fit into a single row

### @comet/admin-color-picker

#### Breaking changes

-   Change the majority of classes used for style overrides. This may break custom styles
-   Remove `showPicker` prop from `ColorPicker` and show the picker by default. Hide by using the `hidePicker` prop

> -   Remove `showClearButton` prop from `ColorPicker`. Re-add it by including the `ClearInputButton` component in the `endAdornment` prop

Reviewer: A clearable prop was added again in https://github.com/vivid-planet/comet/commit/edb181bc9fdcb0f4ecae1f3cd3ab1ca5476e9e53

```suggestion
-   Rename the `showClearButton` prop to `clearable` in `ColorPicker`
```
